cristin-person/src/main/java/no/unit/nva/cristin/person/picture/fetch/FetchPictureApiClient.java

```diff
@@ -0,0 +1,110 @@
+package no.unit.nva.cristin.person.picture.fetch;
+
+import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_BACKEND_FAILED_WITH_EXCEPTION;
+import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_BACKEND_FETCH_FAILED;
+import static no.unit.nva.cristin.common.client.CristinAuthenticator.basicAuthHeader;
+import static no.unit.nva.cristin.model.Constants.BASE_PATH;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE;
+import static no.unit.nva.cristin.model.Constants.DOMAIN_NAME;
+import static no.unit.nva.cristin.model.Constants.HTTPS;
+import static no.unit.nva.cristin.model.Constants.PERSON_PATH;
+import static no.unit.nva.cristin.model.Constants.PERSON_PATH_NVA;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.util.Base64;
+import no.unit.nva.cristin.common.client.ApiClient;
+import no.unit.nva.cristin.person.model.nva.Binary;
+import no.unit.nva.exception.FailedHttpRequestException;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.paths.UriWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FetchPictureApiClient extends ApiClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(FetchPictureApiClient.class);
+
+    public static final String PICTURE_PATH = "picture";
+
+    private final transient HttpClient httpClient;
+
+    public FetchPictureApiClient(HttpClient httpClient) {
+        super(httpClient);
+        this.httpClient = httpClient;
+    }
+
+    /**
+     * Retrieves a person's profile picture in Cristin.
+     *
+     * @return A byte[] containing person profile picture
+     * @throws ApiGatewayException if something went wrong that can be mapped to a client response
+     */
+    public Binary fetchPicture(String personId) throws ApiGatewayException {
+        var uri = generateCristinUri(personId);
+        var response = fetchBinary(uri);
+        checkHttpStatusCode(generateIdUri(personId), response.statusCode());
```

Comment:
Someone once said local vars were redundant and could lead to bugs, another said local vars increased readability. Im not sure what is best, maybe a combination

Reply:
true,

I commented on this because of this 
`checkHttpStatusCode(generateIdUri(personId), response.statusCode());`

so that you use the same style